src/main/java/htsjdk/samtools/SAMTextWriter.java

```diff
@@ -158,9 +158,9 @@ public void writeAlignment(final SAMRecord alignment) {
                 out.write(FIELD_SEPARATOR);
                 final String encodedTag;
                 if (attribute.isUnsignedArray()) {
-                    encodedTag = tagCodec.encodeUnsignedArray(tagUtil.makeStringTag(attribute.tag), attribute.value);
+                    encodedTag = tagCodec.encodeUnsignedArray(SAMTagUtil.makeStringTag(attribute.tag), attribute.value);
                 } else {
-                    encodedTag = tagCodec.encode(tagUtil.makeStringTag(attribute.tag), attribute.value);
+                    encodedTag = tagCodec.encode(SAMTagUtil.makeStringTag(attribute.tag), attribute.value);
```

Comment:
Now that `tagUtil` is unused it should be deleted.

Reply:
did this, good catch